### PR TITLE
MT3-523: #ready-for-test Update radios to use lbh-frontend-react

### PR DESCRIPTION
**Pull request by LBHELewis**
Updates radios to use lbh-frontend-react (requires `npm install`)

### CODE REVIEW

File: components/RadioButtons.tsx [hunk 1]
```diff
@@ -84,9 +54,16 @@ RadioButtons.propTypes = {
...
+    PropTypes.shape({
+      // id: PropTypes.string,
+      value: PropTypes.string.isRequired,
+      label: PropTypes.string.isRequired
+      // label: PropTypes.exact({
+      //   id: PropTypes.string,
+      //   children: PropTypes.node.isRequired
+      // }).isRequired,
+      // checked: PropTypes.bool,
+      // disabled: PropTypes.bool
```
Comment (erbridge) [inline components/RadioButtons.tsx]: Let's clean these up.

File: package-lock.json [hunk 2]
```diff
@@ -8283,14 +8283,12 @@
             "balanced-match": {
               "version": "1.0.0",
               "bundled": true,
-              "dev": true,
-              "optional": true
+              "dev": true
```
Comment (erbridge) [inline package-lock.json]: I suspect this is an error. Run `npm install` again to see if they return.

**Review by erbridge — COMMENTED** [review]
Couple of minor comments.

File: package-lock.json (hunk 2 above)
Comment (LBHELewis) [reply package-lock.json]: ...bettER?

Comment (erbridge) [reply package-lock.json]: Dunno. I'm sure it's fine.

**Review by erbridge — APPROVED** [review]
Please squash before merging.

**Pull request merged**